gobblin-cluster/src/main/java/gobblin/cluster/GobblinHelixJobLauncher.java

```diff
@@ -147,42 +141,6 @@ public GobblinHelixJobLauncher(Properties jobProps, final HelixManager helixMana
 
     this.taskStateCollectorService = new TaskStateCollectorService(jobProps, this.jobContext.getJobState(),
         this.eventBus, this.stateStores.taskStateStore, outputTaskStateDir);
-
-
-
-    // HACK to fixup IDEAL state with a rebalancer that will rebalance running jobs as well
-    final String clusterName = ConfigUtils.getString(jobConfig, GobblinClusterConfigurationKeys.HELIX_CLUSTER_NAME_KEY, "");
-    if (!clusterName.isEmpty()) {
-      this.helixManager.addIdealStateChangeListener(new IdealStateChangeListener() {
-        @Override
-        public void onIdealStateChange(List<IdealState> list, NotificationContext notificationContext) {
-          HelixAdmin helixAdmin = helixManager.getClusterManagmentTool();
-          for (String resource : helixAdmin.getResourcesInCluster(clusterName)) {
-            IdealState idealState = helixAdmin.getResourceIdealState(clusterName, resource);
-            if (idealState != null) {
-              /**
-               * Commenting this out until we fix the rebalancer
-               final String rebalancerClassDesired = GobblinTaskRebalancer.class.getName();
-               if (!idealState.getRebalancerClassName().equals(rebalancerClassDesired)) {
-               idealState.setRebalancerClassName(rebalancerClassDesired);
-               helixAdmin.setResourceIdealState(clusterName, resource, idealState);
-               }
-               **/
-
-              // HACK to ensure that concurrent tasks per instance is set.
-              // TODO: Remove this when we upgrade to a fixed Helix version
-              final String taskResourceName = TaskUtil.getNamespacedJobName(jobContext.getJobName(), jobContext.getJobId());
-              int jobConcurrency = ConfigUtils.getInt(jobConfig, GobblinClusterConfigurationKeys.HELIX_CLUSTER_TASK_CONCURRENCY,
-                  GobblinClusterConfigurationKeys.HELIX_CLUSTER_TASK_CONCURRENCY_DEFAULT);
-              HelixConfigScope resourceScope =
-                  new HelixConfigScopeBuilder(HelixConfigScope.ConfigScopeProperty.RESOURCE).forCluster(clusterName).forResource(taskResourceName).build();
```

Comment:
When we delete this code, do we know if concurrent jobs per instance setting will be honored?